public/doc/swagger-2-v1.0.0.yaml

```diff
@@ -17,6 +17,47 @@ security:
   - UserSecurity: []
 
 paths:
+  /container_projects{container_project_id):
```

Comment:
Should be `/container_projects/{container_project_id}`

Reply:
Good eye, thanks!